climpred/tests/test_stats.py

```diff
@@ -172,6 +174,63 @@ def test_bootstrap_func_multiple_sig_levels(control_3d_NA):
     bootstrap = 5
     sig = [5, 95]
     actual = dpp_threshold(ds, bootstrap=bootstrap, sig=sig)
-    print(actual)
     assert actual['quantile'].size == len(sig)
     assert (actual.isel(quantile=0).values <= actual.isel(quantile=1)).all()
+
+
+# @pytest.mark.skip(reason='dask doesnt work yet on all stats funcs')
+# @pytest.mark.parametrize(
+#    'func', (dpp, varweighted_mean_period, decorrelation_time, autocorr)
+# )
```

Comment:
Are these supposed to be commented out?

Reply:
currently `decorrelation_time` fails. I dont get why. some error with the coordinates when lazy.

Reply:
well I get an error in the tests but to error when running it in my ipython kernel

Reply:
xfailed